src/main/java/com/synopsys/arc/jenkins/plugins/ownership/jobs/OwnershipActionFactory.java

```diff
@@ -39,14 +39,19 @@
  * @author Oleg Nenashev, Synopsys Inc.
  */
 @Extension
-public class OwnershipActionFactory extends TransientProjectActionFactory {
+public class OwnershipActionFactory extends TransientActionFactory<Job> {
```

Comment:
Theoretically yes. I doubt action factory can be used anywhere, but maybe it's a topic for changelogs